### PR TITLE
more fixes to globreg test with old VM

### PR DESCRIPTION
**Pull request by pondrejk**
adding changes I didn't realize were needed in  #8412

### CODE REVIEW

**Comment by JacobCallahan** [comment]
This should go direct into the new 6.9-legacy branch.

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -1377,7 +1377,7 @@ def test_positive_global_registration_end_to_end(
     GlobalParameter().set({'name': 'host_registration_insights', 'value': 1})
     GlobalParameter().set({'name': 'host_registration_remote_execution', 'value': 1})
     # rex interface
-    iface = 'eth0'
+    iface = 'ens3'
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_host.py:1380]: Is this interface applicable to the test.only when using the virtual machine class?

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (pondrejk) [reply tests/foreman/ui/test_host.py:1380]: That's what I figured from the automation logs. Successful PRT run in 8356 tells me eth0 is ok for the master branch.

**Comment by pondrejk** [comment]
@JacobCallahan yes, this PR is open for the 6.9-legacy branch ;)

**Comment by JacobCallahan** [comment]
> @JacobCallahan yes, this PR is open for the 6.9-legacy branch ;)

Ha! You're right!

File: tests/foreman/ui/test_host.py (hunk 1 above)
Comment (JacobCallahan) [reply tests/foreman/ui/test_host.py:1380]: Perfect, thanks for looking into that

**Review by JacobCallahan — APPROVED** [review]

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Pull request merged**